### PR TITLE
fix(dataloaders): fix shuffle logic

### PR DESCRIPTION
**Pull request by parkererickson-tg**

### CODE REVIEW

**Review by qe-tigergraph — COMMENTED** [review]
Unit Test: SUCCESS, e2e Test: SUCCESS, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/667/

**Review by qe-tigergraph — APPROVED** [review]
QE Approved

File: pyTigerGraph/gds/gsql/dataloaders/edge_loader.gsql [hunk 1]
```diff
@@ -35,6 +35,9 @@ CREATE QUERY edge_loader_{QUERYSUFFIX}(
 
     # Shuffle vertex ID if needed
     start = {ANY};
+    IF filter_by IS NOT NULL THEN
```
Comment (billshitg) [inline pyTigerGraph/gds/gsql/dataloaders/edge_loader.gsql:38]: I thought this is removed? because there is filtering by edge on line 60

Comment (parkererickson-tg) [reply pyTigerGraph/gds/gsql/dataloaders/edge_loader.gsql:38]: Good catch - I only removed it in the edge neighbor loader

**Review by qe-tigergraph — COMMENTED** [review]
Unit Test: SUCCESS, e2e Test: SUCCESS, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/668/

**Review by qe-tigergraph — APPROVED** [review]
QE Approved

**Pull request merged**